doorstop/core/publisher.py

```diff
@@ -330,13 +330,12 @@ def _lines_markdown(obj, **kwargs):
                                                   lev=level, u=item.uid)
             else:
                 standard = "{h} {u}".format(h=heading, u=item.uid)
-            attr_list = _format_md_attr_list(item, True)
-            yield standard + attr_list
 
-            # Item Description
             if item.header:
-                yield ""  # break before text
-                yield heading + item.header
```

Comment:
Hi Johnny

The original intent for separating the header from rest of information is for the style to look like this: https://imgur.com/qO9TYhZ

I find that having the section, document ID, and the header all in one line and with the same `<h1>` or `<h2>` style quite awful. I've been on and off working on improving the style here: https://github.com/rickeywang/doorstop/tree/bonus_style_points but it is not ready for PR yet. 

For pure text output I agree with you that having them on one line makes most sense.

Reply:
Hi Rickey,

I still can't see what the benefits are for having the title on two separate lines.  Is this some common practice within requirements documents which I am not aware of?

Specifically, what I don't like about having the two-line title (which is why I created this PR) is:
1. The title in the document has a mismatch from what is seen in the table of contents.  This means you can't visually scan the document, or even use the "find" feature of the browser to find e.g. "2.3 Assets" (because "2.3" and "Assets" are separated)
2. Semantically, having two &lt;h2> tags for one small section makes no sense.  It sort of looks as if there is text missing below the first line.  The style in your upcoming PR seems to correct this somewhat, but I can't really say that I'm a fan of the two-line title. (I find it quite confusing to look at)

What do you think about doing it like this instead:

![doorstop-headers](https://user-images.githubusercontent.com/6860386/36811245-044eac76-1cc5-11e8-81cf-e38b64a84c6a.png)

I think it corrects the "awful" problem, while still being coherent with the table of contents.

Reply:
I think that looks pretty good with the heading in front of the ID.

Reply:
Ah yeah this looks nice. And no, I just made up the separate lines style.

Reply:
Cool! I think we can merge this and let @rickeywang incorporate the additional styling changes later.